guidelines/1.3.1.md

```diff
@@ -1,57 +1,150 @@
-*   Form fields have format requirements or additional information, but it is not properly associated with the corresponding field.
+* Headings are presented using the correct markup or traits;
+* Form fields have properly associated labels;
+* Form fields with format requirements or other additional information have properly associated hints;
+* Tabular data is presented using the correct markup (web only);
+* Lists of items are presented using the correct markup (web only);
+* Sets of radio buttons or checkboxes have a `<fieldset>` element to group them together, and a `<legend>` element to provide a label for the group (web only).
 
-##### Code examples
+### WCAG Success Criteria
 
-###### iOS
+* [1.3.1 Info and relationships (A)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)
+
+## Why?
+
+This ensures that content structures are available to screen reader, screen magnifier and speech recognition tool users.
+
+## How?
+
+### Design
+
+TODO (before merging this PR)
+
+---
+
+### iOS
+
+Setting a view as a heading (e.g. table section headings or screen titles):
 
 ```swift
-// Setting a label as a heading (e.g. table section headings or screen titles)
 headingLabel.isAccessibilityElement = true
 headingLabel.accessibilityTraits = [.heading]
+```
 
-// Setting a custom view as a search field
+Setting a custom view as a search field:
+
+```swift
 customSearchField.isAccessibilityElement = true
 customSearchField.accessibilityTraits = [.searchField]
+```
 
-// Setting a custom view as a tab bar
+Setting a custom view as a tab bar:
+
+```swift
 customTabBar.isAccessibilityElement = true
 customTabBar.accessibilityTraits = [.tabBar]
 ```
 
-> Note: You can combine traits if your element needs it:
+You can combine traits if your element needs it:
 
 ```swift
 view.accessibilityTraits = [.heading, .updatesFrequently]
 ```
 
-* [UIAccessibilityTraits developer reference on apple.com](https://developer.apple.com/documentation/uikit/uiaccessibility/uiaccessibilitytraits "developer.apple.com reference")
+Providing a label for input fields:
+
+```swift
+textField.accessibilityLabel = "First Name"
+```
+
+#### Supporting docs and references
 
+* [UIAccessibilityTraits developer reference](https://developer.apple.com/documentation/uikit/uiaccessibility/uiaccessibilitytraits "developer.apple.com reference")
 * [What iOS Traits Actually Do on deqeue.com](https://www.deque.com/blog/ios-traits/ "dequeue.com article on UIAccessibilityTraits")
 
-##### Useful resources
+---
+
+### Android
+
+Setting a view as a heading (requires API level 28):
+
+```kotlin
+view.isAccessibilityHeading = true
+```
+
+#### Supporting docs and references
+
+* [View developer reference](https://developer.android.com/reference/android/view/View.html#setAccessibilityHeading(boolean) "developer.android.com reference")
+
+---
+
+### Web
```

Comment:
@jfhector can you review this code for me to make sure I haven't done anything silly? Been a while since I've worked with HTML so I'm very rusty.